frontend/src/lib/Buttons.ts

```diff
@@ -118,7 +118,7 @@ export default class Buttons {
       disabled: !useCurrentResource,
       disabledTitle: useCurrentResource ? undefined : 'Select at least one resource to retry',
       id: 'retryBtn',
-      title: 'Retry',
+      title: 'Retry',      
```

Comment:
Please remove extra spaces at end of this line.

Reply:
done